fappdeploy/httptrigger/GHubEventDispatch.cs

```diff
@@ -180,7 +180,12 @@ public static async Task<IActionResult> Run([HttpTrigger(AuthorizationLevel.Func
                     return (ActionResult)new OkObjectResult("Unrecognized event, could not be sent");
                    
             }
-
+            string req_type = req_data.GetType().ToString();
+            if(req_type == "Newtonsoft.Json.Linq.JValue")
+            {
+                String tmp = req_data.ToString();
```

Comment:
are you able to test this properly?

Reply:
yes tested this with actual event and sample event.